### PR TITLE
Adding support for sorting by SchemaTarget.

### LINKED ISSUE #5615: Unable to sort tasks by Schema Target
**Description:**
Using Schema Target as a sort column causes exception.

### PR DESCRIPTION
**Pull request by corneil**
Fixes #5615 and https://github.com/spring-cloud/spring-cloud-dataflow-ui/issues/1948

### CODE REVIEW

File: spring-cloud-dataflow-aggregate-task/src/main/java/org/springframework/cloud/dataflow/aggregate/task/impl/AggregateDataFlowTaskExecutionQueryDao.java [hunk 1]
```diff
@@ -54,6 +38,20 @@
 import org.springframework.util.CollectionUtils;
 import org.springframework.util.StringUtils;
 
+import javax.sql.DataSource;
```
Comment (onobc) [inline spring-cloud-dataflow-aggregate-task/src/main/java/org/springframework/cloud/dataflow/aggregate/task/impl/AggregateDataFlowTaskExecutionQueryDao.java:41]: Either the import order was wrong before, or it is now. Please adjust.

**Review by onobc — CHANGES_REQUESTED** [review]
@corneil I think your Intellij formatter is not setup correctly (import order issues in the PR).

File: spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/TaskExecutionControllerTests.java [hunk 2]
```diff
@@ -637,6 +637,13 @@ void sorting() throws Exception {
 				.andDo(print())
 				.andExpect(status().isOk());
 
+		mockMvc.perform(get("/tasks/executions").param("sort", "SCHEMA_TARGET").accept(MediaType.APPLICATION_JSON))
```
Comment (onobc) [inline spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/TaskExecutionControllerTests.java:640]: This is good that we have added a controller test for this, but do we have anything that actually tests the query and verifies the sort works properly? That is the part that I am more concerned with.

Comment (corneil) [reply spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/TaskExecutionControllerTests.java:641]: Added to `org.springframework.cloud.dataflow.server.repository.TaskExecutionExplorerTests`

**Comment by corneil** [comment]
> @corneil I think your Intellij formatter is not setup correctly (import order issues in the PR).

Installed spring-java-format for IntelliJ.

**Review by onobc — APPROVED** [review]
LGTM @corneil - thanks for the updates. Great work!

**Pull request merged**